Zend/zend_compile.c

```diff
@@ -6535,29 +6499,28 @@ void zend_compile_prop_decl(zend_ast *ast, zend_ast *type_ast, uint32_t flags, H
 			ZVAL_UNDEF(&value_zv);
 		}
 
-		zend_declare_typed_property(ce, name, &value_zv, flags, doc_comment, attributes, type);
+		zend_declare_typed_property(ce, name, &value_zv, flags, doc_comment, type);
```

Comment:
We can change the signature of this function to return the `zend_property_info *` instead.